src/providers/dynamic-plugin.ts

```diff
@@ -33,4 +41,32 @@ export const extensions: EncodedExtension[] = [
     },
     type: 'console.page/resource/list',
   } as EncodedExtension<ResourceListPage>,
+  {
+    properties: {
+      component: {
+        $codeRef: 'ProviderDetailsPage',
+      },
+      model: ProviderModelGroupVersionKind,
+    },
+    type: 'console.page/resource/details',
+  } as EncodedExtension<ResourceDetailsPage>,
+
+  {
+    properties: {
+      model: ProviderModelGroupVersionKind,
+      ...ProviderModel,
```

Comment:
I don't think this spread is needed, can you please check? if not needed also remove this from line 67-68